elections/committee-conflict-of-interest.md

```diff
@@ -0,0 +1,61 @@
+# Committee Conflicts of Interest
+
+**Effective date**: 2023-10-03 (following the announcement of the [2023 Steering Committee election][steering-2023-election-schedule] results)
+
+[steering-2023-election-schedule]: /elections/steering/2023/README.md#schedule
+
+At time of writing, the only known conflict of interest for serving on
+committees in the Kubernetes Community exists with the
+[Steering Committee](/committee-steering/README.md) and
+the [Code of Conduct Committee (CoCC)](/committee-code-of-conduct/README.md).
+
+## Steering Committee and Code of Conduct Committee
+
+**_Kubernetes Community members cannot serve on both committees simultaneously._**
+
+This conflict may arise in the following scenarios:
+
+- A standing Steering Committee member has accepted a nomination to serve on
+  the Code of Conduct Committee
+- A standing CoCC member has elected to run for a seat on the Steering
+  Committee
+- A Kubernetes Community member has been nominated for candidacy for both
+  committees
+
+There are several factors under consideration when formulating a reasonable
+process for resolving this conflict of interest:
+
+- The distinctness of the Steering and CoCC charters, not limited to how these
+  committees need to adjudicate on community matters
+- The Kubernetes Community's expectation of commitment in its committee members
+  for duration of their terms
+- Continuity of committees and the duties they discharge
+- Undue burden on existing committee members to balance workload through
+  committee membership transitions, not limited to onboarding/offboarding
+- Undue burden on Election Officers in supporting overlapping elections with
+  the potential to generate conflicts
+- Minimization of conflict of interest edge cases not otherwise described by
+  this document
+
+With these considerations in mind, the following rules exist to prevent or
+resolve membership conflicts of interest between Steering and the CoCC:
+
+<!-- TODO(committee-coi): Reflect this list in each committee's "Eligibility for candidacy" section -->
+
+- Loss of committee member that occurs due to this conflict of interest will
+  be resolved per existing election or committee vacancy rules
+- Candidates who are nominated for a seat on both Steering and CoCC within the
+  same 12-month period must:
+  - (if elected) accept a seat on the committee whose election concludes first
+  - be withdrawn from candidacy in second election to conclude
+- Standing Steering or CoCC members must have served on their committee for a
+  period of twelve (12) months to be considered eligible for candidacy in the
+  other committee's election
```

Comment:
I like this sentiment, but I see a problem with the number 12 months, specifically in how the current elections are lined up.  If the elections were staggered by 6 months, this would be completely reasonable, but if the elections are close together, as they are right now, and they happen at about the same time every year, this places an unbalanced restriction on the committee that has its elections first.  Whatever the candidates reasons might be for wanting to change committees in the first place (for example: wanting more/less emotional labor), I would suggest this number change to slightly less (10 or 11 months), to not be such a unidirectional disadvantage.  This would also avoid even the perception of any possible artificial forcing function pushing back the annual cycle of elections.

Reply:
As I'm thinking about this, we could also say slightly longer (14 months), and still avoid the imbalance when elections are close on the calendar.